src/components/FAIcon.js

```diff
@@ -25,6 +30,7 @@ FAIcon.propTypes = {
     PropTypes.string,
     PropTypes.elementType,
   ]),
+  extraClassname: PropTypes.string,
```

Comment:
```suggestion
  extraClassName: PropTypes.string,
```
this keep sit consistent with the `className` react prop.